### ISSUE TITLE
amygdala microinfarct

### ISSUE THREAD

**Issue by sarahstofel** [mgijax/mammalian-phenotype-ontology#3422]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

amygdala microinfarct

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

an infarct located in the almond-shaped body of basal nuclei anterior to the inferior horn of the lateral ventricle of the brain, within the temporal lobe that can only be observed microscopically

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

https://jamanetwork.com/journals/jamaneurology/fullarticle/776636
PMID: 10867785

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

cerebral microinfarct, abnormal amygdala morphology

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

https://orcid.org/0000-0003-2307-1226

**Comment by sbello** [comment]
I don't know that this belongs under cerebral microinfarct. I've put it under brain lesion and abnormal amygdala morphology for now. The definition for abnormal amygdala morphology does mention the temporal lobe which would put this in the cerebral hemishphere but the exiting morphology term only has limbic system as a parent. Unsure as to why this is.

Brain Info and EMPA have amygdala as a child of basal ganglia which is a sibling of cerebral hemisphere in EMAP.

In UBERON amygdala is also a child of cerebral hemisphere gray matter.

Will check with Cindy and change the parent if needed.

**Comment by sbello** [comment]
MP:0021007
